packages/fhir/src/Adaptor.js

```diff
@@ -33,21 +42,37 @@ export function execute(...operations) {
  * Creates a resource in a destination system using a POST request
  * @public
  * @example
- * create("Bundle", {
- *   entry: [
- *     {
- *       fullUrl: "", // Eg: Patient URL
- *       resource: {}, // Resource data
- *       search: {
- *         mode: "match",
- *       },
- *     },
- *   ],
- *   type: "collection",
- * });
+ * post("Bundle",{
+ * body:{
+ * "resourceType": "Bundle"
+ * },
+ * parseAs:'json',
+ * headers:{content-type:'application/json'}
+ * })
+ * @function
+ * @param {string} path - Path to resource
+ * @param {RequestOptions} params - contains body object to create the new resource, headers, and throwOnError
```

Comment:
Just nitty picking, not really important

Should we call it `RequestParams` instead of ~RequestOptions~ ?

Reply:
There's something going on here with params and options that I'm not that happy about... Trying to work out a good short term solution 

I actually think that create and post have different signatures

Reply:
I think they are the same well at-least the intention for `create()` / `post()`. I remember when i was working on this we choose to use the word `create` because it's more related to both `developer` and `non-developer`. 

It's the same naming we have in `DHIS2`, the `create()` operation is actually a customized `post()` function which makes it's easy to create different resources

Reply:
@mtuchi What do you think is the difference between a Param and an Option?

Reply:
When i see `params` in my head this is something like `query, body, headers`, where i see options it translate to something like `throwOnError`, `pollTimeout` [Options usually have defaults values]. 

I think both can be used mean the same thing, But usually what goes to my head when i see `options` keyword, I think these are optional params

Reply:
Interesting

It's an open question. There is no formal official definition, and so far as I know there's no strict openfn usage for these terms.

We often use params for our http helpers, but actually params is not conceptually tied to HTTP at all.

I guess options only works if there are no required keys.

It's the kind of terminology I'd like us to be consistent with across adaptors. I don't know the answers yet - just wanted your thoughts.